.github/workflows/health_checks.yml

```diff
@@ -436,6 +436,7 @@ jobs:
   check_package_versions:
     if: github.event_name == 'pull_request'
     runs-on: ubuntu-latest
+    timeout-minutes: 10
```

Comment:
for some reason this check tends to hang when:
1. empty changeset is added
2. head commit on target branch is release commit

This is not a fix, but to make life easier.